src/Dfe.Spi.Common/Dfe.Spi.Common.Caching/Definitions/Managers/ICacheManager{TCacheKey,TManagerItem}.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace Dfe.Spi.Common.Caching.Definitions.Managers
+{
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    /// <summary>
+    /// Describes the operations of the cache manager.
+    /// </summary>
+    /// <typeparam name="TCacheKey">
+    /// The type of key used in the underlying storage.
+    /// </typeparam>
+    /// <typeparam name="TManagerItem">
+    /// The type of item being managed.
+    /// </typeparam>
+    public interface ICacheManager<TCacheKey, TManagerItem>
```

Comment:
As an ICacheManager is basically just decorating an ICacheProvider, can we use the same generic type names? i.e. TCacheValue rather than TManagerItem.

Reply:
Code updated to use strings and objects, as suggested.